### ISSUE TITLE
Add parsing for Gopro *.360 format

### ISSUE THREAD

**Issue by smathermather** [localdevices/ODMax#12]
GoPRO .360 format is weird. Fortunately, David at Trekview did the homework in this 4 part series:
https://www.trekview.org/blog/2021/reverse-engineering-gopro-360-file-format-part-1/
https://www.trekview.org/blog/2021/reverse-engineering-gopro-360-file-format-part-2/
https://www.trekview.org/blog/2021/reverse-engineering-gopro-360-file-format-part-3/
https://www.trekview.org/blog/2021/reverse-engineering-gopro-360-file-format-part-4/

The TL;DR is that it is a cube format already, but split into 2 video channels, with some (36 pixels) overlap between the cube faces in order to allow for blending.
![image](https://user-images.githubusercontent.com/1174901/142709392-682beafe-35c5-4f8e-8ba0-9c8ba5b59a29.png)

**Comment by smathermather** [comment]
An easy way to solve this is with the imagemagick python API replicating this bit of work:
```
# top row
convert img1_cubemap.png cubefaces/left_img1.jpg -geometry +0+0 -composite PNG32:img1_cubemap.png
convert img1_cubemap.png cubefaces/forward_img1.jpg -geometry +1344+0 -composite PNG32:img1_cubemap.png
convert img1_cubemap.png cubefaces/right_img1.jpg -geometry +2688+0 -composite PNG32:img1_cubemap.png
# bottom row
convert img1_cubemap.png cubefaces/bottom_img1.jpg -geometry +0+1344 -composite PNG32:img1_cubemap.png
convert img1_cubemap.png cubefaces/back_img1.jpg -geometry +1344+1344 -composite PNG32:img1_cubemap.png
convert img1_cubemap.png cubefaces/top_img1.jpg -geometry +2688+1344 -composite PNG32:img1_cubemap.png
```

**Comment by smathermather** [comment]
It works! Original image from frame 1 of track 1 (index 0 in ffmpeg):
![img1](https://user-images.githubusercontent.com/1174901/142716511-241591ad-f1f4-49ff-b6a4-bf41c830d92d.jpg)

Then a left face to the cube, without blending:
![left_img1](https://user-images.githubusercontent.com/1174901/142716538-109189d5-a41a-4c44-95f8-be0a97aa1bbd.jpg)

Right face without blending:
![right_img1](https://user-images.githubusercontent.com/1174901/142716551-ac84f7e7-1fcf-43b5-82cd-c9554f43a260.jpg)

Forward doesn't need blending:
![forward_img1](https://user-images.githubusercontent.com/1174901/142716560-eae194e0-5084-447b-92c5-d2daf10e2cf9.jpg)
